### DISCUSSION TITLE
Welcome to WingOS_x64 Discussions!

### DISCUSSION THREAD

**Discussion by Supercip971** [Supercip971/WingOS#8] [General]
# Hello world !
You can discuss anything that is in relation with the WingOS project !

**Comment by circutrider21** [comment]
Have you thought of using a more powerful build system such as [Meson](mesonbuild.com) or [GN](https://gn.googlesource.com/gn/). I could help you if you are intrested.

**Reply by Supercip971** [reply]
you can do it if you wish, and if you don't mind ;)

**Reply by circutrider21** [reply]
ok!
